src/DurableTask.Netherite/OrchestrationService/PartitionManagementOptions.cs

```diff
@@ -18,13 +18,20 @@ public enum PartitionManagementOptions
         ClientOnly,
 
         /// <summary>
-        /// Use the event processor host implementation provided by the EventHubs client library.
+        /// Use the event processor host implementation provided by the EventHubs client library. This dynamically
+        /// balances the partitions across all connected hosts.
         /// </summary>
         EventProcessorHost,
 
         /// <summary>
         /// Follow a predefined partition management script. This is meant to be used for testing and benchmarking scenarios.
+        /// This was an internal feature and is no longer supported.
         /// </summary>
         Scripted,
```

Comment:
nit: I'm curious if we can mark this as obsolete to prevent future usage.

```suggestion
        /// <summary>
        /// Follow a predefined partition management script. This is meant to be used for testing and benchmarking scenarios.
        /// This was an internal feature and is no longer supported.
        /// </summary>
         [Obsolete]
        Scripted,
```

Reply:
marking it as obsolete may violates major-minor version change rules.